website/docs/reference/dbt-jinja-functions/log.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "log"
 id: "log"
-description: "Use `log` to log a line to either the log file or stdout."
+description: "Read this guide to understand the log Jinga function in dbt."
```

Comment:
`Jinga` should be `Jinja`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/log

content="Read this guide to understand the log Jinja function in dbt."